Makefile

```diff
@@ -99,4 +99,3 @@ debug.stop: ## Run docker-compose with debug
 
 swag:
 	@swag init
-	@swag fmt
```

Comment:
if we don't format, it won't update files. would make `make swag` useless, no?

Reply:
Nah it was causing issues. Fmt just created weird broken diffs

Reply:
swag fmt only formats your go code

Reply:
>    (optional) Use swag fmt format the SWAG comment. (Please upgrade to the latest version)

Reply:
I think we had an edge case with the formatter where it wasn't formatting correctly and adding a tab each time for some reason